Assets/Scripts/Gamemode/RacingGamemode.cs

```diff
@@ -8,12 +8,27 @@ namespace Assets.Scripts.Gamemode
 {
     public class RacingGamemode : GamemodeBase
     {
+
+        /// <summary>
+        /// issue #277 will have to be removed once the conversion from fenggamemanager to single manager will be done
+        /// </summary>
+        public const int IntRoundTimeStatusTopScale = 10;
```

Comment:
Use [Obsolete] instead of a summary if you intention is that this should not be used in the future

Reply:
yeah, just didn't want to put new stuff directly as obsolete :'D

Reply:
Also cos it wouldn't be atm as the replacement code is not there yet